### ISSUE TITLE
Will support webview2?

### ISSUE THREAD

**Issue by Losses** [patr0nus/DeskGap#42]
It'll would be great if this lib support Webview2 :)
https://docs.microsoft.com/en-us/microsoft-edge/hosting/webview2

**Comment by patr0nus** [comment]
Yes. Chromium Edge's WebView2 is definitely something I want DeskGap to support. Stay tuned!

**Comment by tmm1** [comment]
A redistributable WebView2 runtime will soon be available for Win7 and above: https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution
